### PR TITLE
New navigations

### PR DESCRIPTION
**Pull request by jasny**
@Nikulio Reapply of PR #117

### CODE REVIEW

**Comment by jasny** [comment]
**Not ready to merge.**

screenshot-legalthings one-2018 06 13-10-11-03 1
screenshot-legalthings one-2018 06 13-10-11-03

**Comment by jasny** [comment]
When scrolling the menu, should go from `static` position to `fixed` position with a semi-transparent background. This should work **exactly the same** as the nav bar on [legalfling.io](https://legalfling.io).

screenshot-legalthings one-2018 06 13-11-28-17

Remove the old navbar.

**Comment by jasny** [comment]
The nav for the mobile version should just be as it is now.

**Comment by Nikulio** [comment]
Updated this pr, merged with my prev branch. Waiting for feedback. Cheers.

**Comment by jasny** [comment]
@Nikulio There is a huge merge conflict.

File: index.html [hunk 1]
```diff
@@ -178,6 +178,64 @@
             <section id="hero"
                      class="hero-full parallax-section svg-custom-hero"
                      data-parallax-image="img/hero-solution.png" data-parallax-opacity="-0.5">
+	
+	            <div class="hero-nav hero-nav--mobile">
+		            <div class="hero-nav__menu">
+			            <a href="#page-title" class="hero-nav__menu-list">Home</a>
+			            <a href="#what-is-legal" class="hero-nav__menu-list">Documentation</a>
```
Comment (jasny) [inline index.html]: Documentation is on https://docs.livecontracts.io

File: index.html [hunk 2]
```diff
@@ -178,6 +178,64 @@
...
+			            <a href="#our-clients" class="hero-nav__menu-list">Playground</a>
```
Comment (jasny) [inline index.html]: Playground isn't available yet, please disable this menu option (grayed out) with `href="javascript:void"`

File: index.html [hunk 3]
```diff
@@ -178,6 +178,64 @@
...
+			            <a href="#in-the-press" class="hero-nav__menu-list">Whitepaper</a>
```
Comment (jasny) [inline index.html]: Needs to point to 'Discover LegalThings One' (where ppl can find the whitepapers).

**Pull request merged**